src/main/fc/rc.c

```diff
@@ -572,55 +567,66 @@ FAST_CODE_NOINLINE void calculateFeedforward(const pidRuntime_t *pid, int axis)
 
-    float feedforward = setpointSpeed + setpointAcceleration;
+    // apply jitter attenuation to classic feedforward elements only (twice on acceleaertion)
+    feedforward *= jitterAttenuator;
 
-    if (axis == FD_ROLL) {
-        DEBUG_SET(DEBUG_FEEDFORWARD, 2, lrintf(feedforward * 0.1f));
-        // un-smoothed feedforward including acceleration but before limiting, transition, averaging, and jitter reduction
-    }
-
-    // apply feedforward transition
+    // apply feedforward transition to classic feedforward elements only
     const bool useTransition = (pid->feedforwardTransition != 0.0f) && (rcDeflectionAbs[axis] < pid->feedforwardTransition);
     if (useTransition) {
         feedforward *= rcDeflectionAbs[axis] * pid->feedforwardTransitionInv;
     }
 
-    // apply averaging
-    if (feedforwardAveraging) {
-        feedforward = laggedMovingAverageUpdate(&feedforwardDeltaAvg[axis].filter, feedforward);
+    // apply max rate limiting, forcing to zero when sticks approach max, but not on yaw
+    if (axis < FD_YAW) {
+        if (pid->feedforwardMaxRateLimit) {
+            if (feedforward * setpoint > 0.0f) { // in same direction
+                const float limit = (maxRcRate[axis] - fabsf(setpoint)) * pid->feedforwardMaxRateLimit;
+                feedforward = (limit > 0.0f) ? constrainf(feedforward, -limit, limit) : 0.0f;
+            }
+        }
+    } else {
+        // add high-pass filtered setpoint to feedforward for yaw
+        // this is not a derivative, being proportional to setpoint, so does not require jitter reduction
+        // it is not interpolated, so dropouts will cause flat spots, but no derivative drops to zero
+        // this provides a sustained FF on yaw that mimics the normal yaw motor drive requirements
+        static float prevSetpointYaw = 0.0f;
+        // const of 7.0 to get decay roughly right
+        const float setpointLpfYaw = prevSetpointYaw + 7.0f * rxInterval * (setpoint - prevSetpointYaw);
+        prevSetpointYaw = setpointLpfYaw;
+        // const of 20 to scale yaw setpoint factor to a decent number
+        feedforward += 20.0f * (setpoint - setpointLpfYaw);
     }
 
-    // apply jitter reduction
-     feedforward *= jitterAttenuator;
+    if (axis == FD_ROLL) {
+        DEBUG_SET(DEBUG_FEEDFORWARD, 2, lrintf(feedforward * 0.1f));
+        // feedforward including acceleration but before averaging
+    }
 
-    // apply max rate limiting
-    if (pid->feedforwardMaxRateLimit && axis < FD_YAW) {
-        if (feedforward * setpoint > 0.0f) { // in same direction
-            const float limit = (maxRcRate[axis] - fabsf(setpoint)) * pid->feedforwardMaxRateLimit;
-            feedforward = (limit > 0.0f) ? constrainf(feedforward, -limit, limit) : 0.0f;
-        }
+    // apply averaging to final values, for additional smoothing if needed
+    if (feedforwardAveraging) {
+        feedforward = laggedMovingAverageUpdate(&feedforwardDeltaAvg[axis].filter, feedforward);
```

Comment:
(laggedMovingAverage is quite poorly performing FIR filter. Even PTx would be better)

Reply:
Yes, indeed it is a poor filter.

However some Rx, eg FrSky, often transmit data in pairs, eg one value high, one low, one high.  Most likely some interaction between sampling and transmit time.  A two-point moving average is optimal for fixing this.  
Likewise the old Crossfire 150Hz mode had two small steps then one big one, reliably in cycles of three.  A 3-point moving average was optimal from removing that wobble.

Hence probably it should continue to exist, for those purposes, and is referenced in Presets specifically for those Rx.